notebook-instance.tf

```diff
@@ -19,14 +19,17 @@ resource "google_notebooks_instance" "notebook_instance_vm" {
     image_family = var.vm_image_image_family
   }
 
-  accelerator_config {
-    core_count = var.accelerator_config_core_count
-    type       = var.accelerator_config_type
+  dynamic "accelerator_config" {
```

Comment:
Nice.  Did you check that the notebook instance is idempotent?  That's a problem I had with dynamic blocks in the past.

Reply:
Hmmm, no I didn't check that. I'll try it out!

Reply:
It seems to test out ok. I've done the following...

1) Create new notebook config in eh-notebooks.tf for notebook 1 instance
2) Terraform plan and apply by specifying notebook 1 as the target
3) Create new notebook config in eh-notebooks.tf for notebook 2 instance
4) Terraform plan and apply by specifying notebook 2 as the target
5) Terraform plan with notebook 1 as the target - no changes needed
6) Terraform plan with notebook 2 as the target - no changes needed
7) Create new notebook config in eh-notebooks.tf for notebook 3 instance
8) Run terraform plan across everything - the plan shows that notebook 1 and 2 will be left as is. Notebook 3 will be created.